tools/cli/commands.go

```diff
@@ -1336,3 +1340,23 @@ func getWorkflowStatus(statusStr string) s.WorkflowExecutionCloseStatus {
 		"[completed, failed, canceled, terminated, continueasnew, timedout]"))
 	return 0
 }
+
+func getWorkflowIdReusePolicy(definedPolicy string) *s.WorkflowIdReusePolicy {
+	if definedPolicy != "" {
+		value, err := strconv.Atoi(definedPolicy)
+		if err != nil {
+			ErrorAndExit(fmt.Sprintf("Option %v format is not supported.", FlagWorkflowIdReusePolicy), err)
+			return nil
+		}
+		policy := s.WorkflowIdReusePolicy(value)
+		for _, p := range s.WorkflowIdReusePolicy_Values() {
```

Comment:
Not sure if there is better way to check value in array

Reply:
To validate input, use `value >= 0 && value < len(s.WorkflowIdReusePolicy_Values())` then you can directly use policy. 
Other way is to use `switch policy ... `.

Reply:
Good point